src/base/anim/controllable.h

```diff
@@ -16,6 +16,10 @@ class Controllable
 
 	Duration getDuration() const { return duration; };
 
+	virtual std::shared_ptr<void> actualPlot() const {
```

Comment:
Codes in the base folder planned to be generic helper codes. This class is a generic controllable animation object which should not know about plots. If we need this here, then I suggest to call it data() or some other generic name.

Reply:
renamed to data